extensions/resume.md

```diff
@@ -78,9 +78,9 @@ This message, sent from a server to a client, indicates that a `RESUME` request
 #### `RESUMED` Message
 This message is sent by the server to indicate that another client has reconnected:
 
-    :nick!olduser@oldhost RESUMED <user> <host> [timestamp]
+    :nick!olduser@oldhost RESUMED <host> [timestamp]
 
-`<nick>`, `<olduser>`, and `<oldhost>` indicate the client that has reconnected, and are the details of the old client. `<user>` and `<host>` indicate the reconnecting client's new username and hostname, and the receiving client MUST process this information as they would a regular [`CHGHOST`](https://ircv3.net/specs/extensions/chghost-3.2.html) message. `<timestamp>`, if given, is a timestamp of the form described above which indicates the last message received by the reconnecting clent.
+`<nick>`, `<olduser>`, and `<oldhost>` indicate the client that has reconnected, and are the details of the old client. `<host>` indicates the reconnecting client's new hostname, and the receiving client MUST process this information as they would a regular [`CHGHOST`](https://ircv3.net/specs/extensions/chghost-3.2.html) message. `<timestamp>`, if given, is a timestamp of the form described above which indicates the last message received by the reconnecting clent.
```

Comment:
Can you please also replace _"as they would a regular"_ with _"as they would **from** a regular"_

Reply:
Oh, also `<olduser>` with just `<user>`, since there's no... old username.